src/bob/api/health_check.clj

```diff
@@ -0,0 +1,49 @@
+;   This file is part of Bob.
+;
+;   Bob is free software: you can redistribute it and/or modify
+;   it under the terms of the GNU General Public License as published by
+;   the Free Software Foundation, either version 3 of the License, or
+;   (at your option) any later version.
+;
+;   Bob is distributed in the hope that it will be useful,
+;   but WITHOUT ANY WARRANTY; without even the implied warranty of
+;   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+;   GNU General Public License for more details.
+;
+;   You should have received a copy of the GNU General Public License
+;   along with Bob. If not, see <http://www.gnu.org/licenses/>.
+
+(ns bob.api.health-check
+  (:require [hugsql.core :as sql]
```

Comment:
this namespace is unused?

Reply:
Are you asking, if sql is not used? Because line 25 uses sql...

Reply:
ah yes. missed that.